src/exampleTypes/TagsQuery/ActionsMenu.js

```diff
@@ -1,57 +1,57 @@
 import React from 'react'
 import _ from 'lodash/fp'
-import F from 'futil-js'
 import { observer } from 'mobx-react'
 import TagsJoinPicker from '../TagsJoinPicker'
 import { withTheme } from '../../utils/theme'
 import { Flex } from '../../greyVest'
-import { copyTags, tagTerm } from './utils'
+import { copyTags, tagTerm, onClickWrapper } from './utils'
 
-let ActionsMenu = ({ node, tree, open, theme: { Button, Checkbox } }) => (
-  <Flex
-    style={{ minWidth: 240, padding: 10 }}
-    className="tags-query-actions-menu"
-    column
-    justifyContent="stretch"
-    alignItems="stretch"
-  >
-    {!!_.get('tags.length', node) && (
-      <>
-        <Button
-          onClick={() => {
-            copyTags(node)
-            F.off(open)()
-          }}
-        >
-          Copy {_.startCase(tagTerm)}s
-        </Button>
-        <Button
-          style={{ margin: '10px 0' }}
-          onClick={() => {
-            tree.mutate(node.path, {
-              tags: [],
-            })
-            F.off(open)()
-          }}
-        >
-          Clear {_.startCase(tagTerm)}s
-        </Button>
-        <div className="line-separator" />
-      </>
-    )}
-    <label className="labeled-checkbox" style={{ margin: '10px 0' }}>
-      <Checkbox
-        htmlId="stemming"
-        checked={!node.exact}
-        onChange={e => tree.mutate(node.path, { exact: !e.target.checked })}
-      />
-      <span>Include word variations</span>
-    </label>
-    <div>
-      <TagsJoinPicker node={node} tree={tree} />
-    </div>
-  </Flex>
-)
+let ActionsMenu = ({
+  node,
+  tree,
+  open,
+  theme: { Button, Checkbox },
+  onClick = _.noop,
+}) => {
+  let wrapper = onClickWrapper(node, onClick, open)
+  return (
+    <Flex
+      style={{ minWidth: 240, padding: 10 }}
+      className="tags-query-actions-menu"
+      column
+      justifyContent="stretch"
+      alignItems="stretch"
+    >
+      {!!_.get('tags.length', node) && (
+        <>
+          <Button onClick={() => wrapper('copy', () => copyTags(node))}>
```

Comment:
I think we can simplify this if we instead just take a wrapper and assume that the return values are success messages and errors are the error messages. On the user side, we'd have one of these that just logs that with toastify:

https://runkit.com/daedalus28/toasty-futil-aspect

Reply:
Changes done.